code/_onclick/telekinesis.dm

```diff
@@ -171,7 +172,11 @@
 	else
 		. = TRUE
 		apply_focus_overlay()
-		focus.throw_at(target, 10, 1,user)
+		//Only items can be thrown 10 tiles everything else only 1 tile
+		if (isitem(focus))
+			focus.throw_at(target, 10, 1,user)
+		else
+			focus.throw_at(target, 1, 1,user)
```

Comment:
I'd rather this be a var on /atom/movable.

Reply:
That is properly the cleanest way of doing this, created tk_throw_range for this